Getting-Started/Data/Scheduled-Publishing/index-v8.md

```diff
@@ -0,0 +1,58 @@
+---
+versionFrom: 8.0.0
+---
+
+# Scheduled Publishing
+
+Each document in Umbraco can be scheduled for publishing and unpublishing on a pre-defined date and time. 
+
+You can find the options to do this click on the arrow next to the **Save and publish** button and pick **Schedule...*. 
+
+![Scheduled publishing](images/schedule.png)
+
+This will open a modal where you can specify dates and time.
+
+![Scheduled publishing](images/Scheduled-publishing-8.png)
```

Comment:
```suggestion
![Scheduled publishing](images/scheduled-publishing-8.png)
```

Reply:
This one means Github couldn't find the picture - on Our it should work as its not case sensitive, but if we ever change the underlying tooling at some point we may as well have this fixed 😁